scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,10 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
 TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = abspath(join(dirname(__file__), '..', 'templates'))
```

Comment:
`TEMPLATES_DIR` looks in your project, but `TEMPLATES_DIR_BASE` should do inside scrapy module.
`TEMPLATES_DIR_BASE = join(scrapy.__path__[0], 'templates')`